cli/commands/run/command.go

```diff
@@ -0,0 +1,63 @@
+// Package run contains the logic for interacting with OpenTofu/Terraform.
+package run
+
+import (
+	"strings"
+
+	"github.com/gruntwork-io/go-commons/collections"
+	"github.com/gruntwork-io/terragrunt/internal/cli"
+	"github.com/gruntwork-io/terragrunt/internal/errors"
+	"github.com/gruntwork-io/terragrunt/internal/experiment"
+	"github.com/gruntwork-io/terragrunt/options"
+	"github.com/gruntwork-io/terragrunt/tf"
+)
+
+const (
+	CommandName = "run"
+)
+
+func NewCommand(opts *options.TerragruntOptions) *cli.Command {
+	return &cli.Command{
+		Name:        CommandName,
+		Usage:       "Run an OpenTofu/Terraform command. Shortcuts for common `run` commands are provided below.",
```

Comment:
> Shortcuts for common `run` commands are provided below.

Is this true? I don't see any obvious place where the help shows the shortcuts.

Reply:
Took from the RFC

![PUBLIC Terragrunt CLI Redesign 2025-02-06 at 10 31 52 PM](https://github.com/user-attachments/assets/57dc6232-bc3c-4b7c-9f1e-e94db24c88ea)

Reply:
Note that the help in the RFC actually has the shortcuts below it, though (e.g. plan, apply).

When I check the help locally for this branch it doesn't.